.gitignore

```diff
@@ -15,4 +15,5 @@ client/public/assets
 dump.rdb
 .env
 .env.ignore
-node_modules
\ No newline at end of file
+node_modules
+scripts/tmp
```

Comment:
The generated CSVs go in a tmp folder -- I added this just in case someone accidentally runs the script and the CSV didn't delete itself for whatever reason.